lib/oauth2/model.rb

```diff
@@ -30,10 +31,18 @@ def self.duplicate_record_error?(error)
       DUPLICATE_RECORD_ERRORS.any? { |re| re =~ error.message }
     end
 
-    # This will only return an authorisation for non-JWT tokens
-    def self.find_access_token(access_token)
-      return nil if access_token.nil?
-      Authorization.find_by_access_token_hash(Lib::SecureCodeScheme.hashify(access_token))
+    # This will only return an authorisation for JWT and non-JWT tokens
+    def self.find_access_token(token)
```

Comment:
I don't like how token is an array here, could it instead be 2 named arguments: `token_type` and `token` or even better, could this method be split into `find_access_token` and `find_jwt`